web/app/root.tsx

```diff
@@ -5,9 +5,14 @@ import {
   Scripts,
   ScrollRestoration,
 } from "@remix-run/react"
-import "./tailwind.css"
+import { MantineProvider } from "@mantine/core"
+import "@mantine/core/styles.css"
+import { Header } from "./components/header/header"
+import { MobileHeader } from "./components/header/mobileHeader"
+import { useMediaQuery } from "@mantine/hooks"
 
 export function Layout({ children }: { children: React.ReactNode }) {
+  const isMobile = useMediaQuery("(max-width: 1199px)")
```

Comment:
This might be nice to store in a Context, so then individual components can also pull it out when needed (Including the header)